mail-client/claws-mail/claws-mail-3.11.1-r1.ebuild

```diff
@@ -99,6 +99,10 @@ RDEPEND="${COMMONDEPEND}
 	app-misc/mime-types
 	x11-misc/shared-mime-info"
 
+src_prepare() {
+	epatch "${FILESDIR}/${P}_RSSyl-encodings-fix.patch"
```

Comment:
Sorry for checking this late but this is wrong. You're overriding autotools-utils_src_prepare. Which also provides support for `PATCHES` to avoid this.